tests/test_discovery.py

```diff
@@ -21,23 +21,21 @@
 Unit tests for objects created from discovery documents.
 """
 from __future__ import absolute_import
-import six
 
 __author__ = "jcgregorio@google.com (Joe Gregorio)"
 
-from six import BytesIO, StringIO
-from six.moves.urllib.parse import urlparse, parse_qs
-
 import copy
 import datetime
 import httplib2
+import io
 import itertools
 import json
 import os
 import pickle
 import re
 import sys
 import unittest2 as unittest
+import urllib
 from collections import defaultdict
```

Comment:
This line should sort above `import copy` above.

Reply:
Done in 4df9862